server/logging/logger.go

```diff
@@ -200,6 +201,14 @@ func NewNoopLogger(t *testing.T) SimpleLogging {
 	}
 }
 
+// NewNoopLogger creates a logger instance that discards all logs and never
+// writes them. Used for testing.
+func NewNoopCtxLogger(t *testing.T) Logger {
+	return &logger{
+		LoggerFacade: logur.NoopLogger{},
```

Comment:
What does this NoopLogger do btw? The noop logger we have is nice because it logs all the output to `t` so you can at least debug failing tests.

Reply:
It literally did nothing, changed to mimic our `NoopLogger`